### PR TITLE
Add Resque.

### PR DESCRIPTION
**Pull request by gphat**
### What's this PR do?
This adds minimal Resque metrics integration to Datadog.

### Motivation
Internal tools asked!

### Notes
There's more we could do here, but I only did the minimum that was easy since that's what @preinheimer asked for. We can add more and iterate if we like!

### How To Deploy
* Merge
* Add to our internal puppet, etc.

### How to Measure Success
Look for `resque.worker_count`!

### How to Rollback
Revert, merge, repeat `How To Deploy`

cc @preinheimer

### CODE REVIEW

**Comment by gphat** [comment]
It works!
```
2016-05-26 20:47:34,341 | WARNING | dd.collector | checks.disk(disk.py:76) | Using `use_mount` in datadog.conf has been deprecated in favor of `use_mount` in disk.yaml
2016-05-26 20:47:34,357 | INFO | dd.collector | config(config.py:899) | initialized checks.d checks: ['network', 'stripe_consul', 'process', 'ntp', 'resque', 'linux_proc_extras', 'dns_check', 'linux_vm_extras', 'file', 'redisdb', 'disk']
2016-05-26 20:47:34,357 | INFO | dd.collector | config(config.py:900) | initialization failed checks.d checks: []
2016-05-26 20:47:34,357 | INFO | dd.collector | checks.collector(collector.py:533) | Running check resque
Metrics:
[('resque.jobs_failed',
  1464295654,
  '5',
  {'hostname': 'qa-supportbox1.northwest.stripe.io', 'type': 'gauge'}),
 ('resque.queues_count',
  1464295654,
  1,
  {'hostname': 'qa-supportbox1.northwest.stripe.io', 'type': 'gauge'}),
 ('resque.worker_count',
  1464295654,
  1,
  {'hostname': 'qa-supportbox1.northwest.stripe.io', 'type': 'gauge'}),
 ('resque.jobs_processed',
  1464295654,
  '20',
  {'hostname': 'qa-supportbox1.northwest.stripe.io', 'type': 'gauge'})]
Events:
[]
Service Checks:
[]
Service Metadata:
[{}]
    resque
    ------
      - instance #0 [OK]
      - Collected 4 metrics, 0 events & 0 service checks
```

**Comment by preinheimer** [comment]
Huzzah!

:shipit:

File: checks.d/resque.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        'resque:workers': 'resque.worker_count',
+    }
+
+    def check(self, instance):
+        tags = instance.get('tags', [])
+
+        host = instance.get('host')
+        port = instance.get('port')
+
+        conn = redis.Redis(host=host, port=port)
```
Comment (antifuchs) [inline checks.d/resque.py]: One thing that I have found useful is issuing a service check here if the connection fails.

**Comment by antifuchs** [comment]
Looks great to me - I think we'll want to add that service check, but

LGTM

As a quick win!

**Comment by gphat** [comment]
Agreed. I'll make sure and follow up with service checks, tests and other iterative improvements.

**Pull request closed without merging**